src/install-update/index.ts

```diff
@@ -2,10 +2,10 @@ import { uploadModal } from "../legacy";
 
 const preload = () => import("./install-update-dialog");
 
-const startInstallUpdate = async (ev: Event) => {
+export const openInstallDialog = async (filename: string) => {
   preload();
   const dialog = document.createElement("esphome-install-dialog");
```

Comment:
Do we have to create a new dialog element every time, or should we re-use it? Do we also remove the dialog when we close it?

Reply:
Yes we do.